packages/runtime-core/__tests__/scheduler.spec.ts

```diff
@@ -246,12 +246,20 @@ describe('scheduler', () => {
     const job2 = () => {
       calls.push('job2')
     }
-    // queue both jobs
+    const job3 = () => {
+      calls.push('job3')
+    }
+    const job4 = () => {
+      calls.push('job4')
+    }
+    // queue all jobs
     queueJob(job1)
     queueJob(job2)
+    queueJob(job3)
+    queuePostFlushCb(job4)
     expect(calls).toEqual([])
     await nextTick()
     // job2 should be called only once
```

Comment:
Shouldn't this comment be updated to: `// job4 ...`?

Reply:
job1 synchronously invokes job2 (and invalidates the existing job2 in the queue), so it is indeed checking job2.

job3 and 4 are there to ensure the invalidation didn't accidentally terminate the loop early.